### DISCUSSION TITLE
GitHub student verification image too small

### DISCUSSION THREAD

**Discussion by fquerdasi** [community/community#36810] [GitHub Education]
Hi, 

I am trying to apply for the GitHub student package. I have a transcript that shows I am currently enrolled, but I cannot upload it to the verification portal. The transcript sent to me from my university is a PDF, which the portal does not accept. I tried 1) taking a screenshot of the relevant page and converting that from png to jpg, and 2) converting the PDF pages with Acrobat from PDF to jpg. Either way, the GitHub portal states that the image is too small to be used. What else can I do to convert the file to an acceptable format?

**Comment by ettaboyle** [comment]
HI @fquerdasi 👋🏻  Have you seen the info in this post: **[Student Developer Pack Application & FAQs ➡️ Start Here](https://github.com/orgs/community/discussions/17814)**

This blog (linked in that post) has some helpful information too: [How to get the GitHub Student Developer Pack without a student ID](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/)

**Comment by kude5n1k** [comment]
just try to increase the size of the image through any online service

**Comment by jasminet2001** [comment]
Use a website like: https://www.imagediamond.com/image-size-increase/
this increases the size of your image, and it worked for me.

**Reply by OstiumAxioma** [reply]
Dont think it will work, you need to take a live photo on the website with your pc cam, not uploading a taken photo. Not sure why github do this.

**Reply by StephenB-24** [reply]
I can confirm that this worked for me.
